pages/home.html

```diff
@@ -5,8 +5,8 @@
 ---
 
 <section class="nz-grid-full nz-flex">
-  <div class="hero">
-    <div class="vertical">
+  <div class="hero" role="main">
+    <div class="vertical" role="main">
```

Comment:
Is it okay to have more than one role: main ?

Reply:
I've no idea, just that they both link to the same main section :slightly_frowning_face: